Installing-and-Configuring/install-os.md

```diff
@@ -0,0 +1,326 @@
+ 'u' before the normal boot)
+3. Select the install boot option for your distro of choice and work through the
+ standard OS installation procedures (including setting up LUKS disk encryption
+ if desired)
+4. Reboot and your new boot options should be available to be chosen by
+ selecting 'm' at the boot screen
+
+If you want to set a default option so that you don't have to choose at every
+ boot, you can do so from the menu by selecting 'd' on the confirmation screen.
+ You will also be able to seal your disk encryption key using the TPM allowing
+ you to use ensure only a boot password and the proper PCR state can unlock this
+ yet.
+
+(\*) Ubuntu/Debian Note: These systems don't read `/etc/crypttab` in their
+ initrd, so you need to adjust the crypttab in the OS and `update-initramfs -u`
+ to have it attempt to use the injected key.  Due to oddities in the cryptroot
+ hooks, you also need keyscript to be in `/etc/crypttab` even as a no-op
+ `/bin/cat`:
+
+`sda5_crypt UUID=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX /secret.key luks,keyscript=/bin/cat`
+
+(Credit to [http://www.pavelkogan.com/2015/01/25/linux-mint-encryption/](http://www.pavelkogan.com/2015/01/25/linux-mint-encryption/)
+ for this trick).
+
+Installing Qubes
+===
+
+![Heads splash screen]({{ site.baseurl }}/images/Heads_splash_screen.jpg)
+
+Plus in the USB stick with the R4.0 install media into one of the USB3 ports (on
+ the left side of the x230) and boot into USB mode (hit 'u' at the prompt),
+ then boot using this option:
+
+```text
+2. Install Qubes R4.0 [kernel /isolinux/xen.gz console=none]
+```
+
+If that completes with no errors it will launch the Xen hypervisor from the
+ x230's ROM image and start the Qubes installer.  The first few seconds are run
+ with an archaic video mode, so things appear a little weird, but once the dom0
+ kernel initializes the graphics it should look right.
+
+![Qubes partitioning]({{ site.baseurl }}/images/Qubes_partitioning.jpg)
+
+Use default QubesOS partitioning scheme for QubesOS 4.x
+
+![Disk encryption recovery key]({{ site.baseurl }}/images/Disk_encryption_recovery_key.jpg)
+
+The disk encrypt password that you enter here will be used as the
+"recovery password" later.  It should be a long value since you won't
+have to enter it very often; only when upgrading the Heads firmware
+or if there is a need to recover the disk on an external machine.
+You will need it again shortly, so don't lose it yet.
+
+![Signing Qubes binaries in /boot]({{ site.baseurl }}/images/Signing_Qubes_binaries_in__boot.jpg)
+
+Once Qubes has finished installing, you'll need to reboot and select the 'Boot
+ menu' option by hitting 'm'.
+
+Select the first boot option:
+
+```text
+1. Qubes, with Xen hypervisor [...]
+```
+
+Then make this the default boot entry by hitting 'd'.  This will also allow you
+ to seal the disk encryption key.
+
+You will need to input the disk recovery key here (almost for the last time),
```

Comment:
Might want to review Keys.md vocabulary and use caps first letter so users understand we are talking about secrets.

Disk Recovery Key passphrase
Disk Unlock Key passphrase etc
USB Security dongle
USB Security dongle's admin PIN etc
(Even if I disagree with PIN and should be passphrases everywhere to not confuse users any deeper)